resources/[system]/chat/html/Message.js

```diff
@@ -0,0 +1,65 @@
+Vue.component('message', {
+  template: '#message_template',
+  data() {
+    return {};
+  },
+  computed: {
+    textEscaped() {
+      let s = '';
+      if (this.template) {
+        s = this.colorize(this.template);
+      } else {
+        s = this.colorize(this.templates[this.templateId]);
+      }
+      return s.replace(/{(\d+)}/g, (match, number) => {
+        const argEscaped = this.args[number] != undefined ? this.escape(this.args[number]) : match
+        if (number == 0 && this.color) {
+          //color is deprecated, use templates or ^1 etc.
```

Comment:
Yea, [colorizeOld](https://github.com/citizenfx/cfx-server-data/pull/1/files/2e075e0651d72e505d31e1546a826c52acd78a2b#diff-90f2afbded91af442406def394fb16e7R25) will only color the first argument if a color is present. [colorize](https://github.com/citizenfx/cfx-server-data/pull/1/files/2e075e0651d72e505d31e1546a826c52acd78a2b#diff-90f2afbded91af442406def394fb16e7R28) will replace `^1` with the color classes in the whole template. It's just important to not use `^1` inside the args, but in the template. This prevents users from injecting colors into their message. Is this desired behavior?